<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tuning</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Vysor</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>150</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cursor X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_csrX">
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Cursor Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_csrY">
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_x1">
            <property name="maximum">
             <number>1920</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>y1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_y1">
            <property name="maximum">
             <number>1080</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_x2">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1920</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>y2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_y2">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1080</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>141</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_auto">
          <property name="text">
           <string>Autoget Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_show_autoget">
          <property name="text">
           <string>Show Autoget Window</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Manual Detect</string>
      </property>
      <widget class="QPushButton" name="btn_party_HP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Party HP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_enemy_party">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Enemy Party</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_drop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Drop</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>360</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Solenoid</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>13</y>
         <width>261</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Period (s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spb_period">
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_touch">
          <property name="text">
           <string>Touch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Motor Control</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>[Motor 2] Steps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>[Motor 1] Steps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>[Motor 2] Goto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>[Motor 1] Goto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>[Motor 2] GotoX</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>[Motor 1] GotoY</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>55</width>
         <height>152</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSpinBox" name="spb_steps2">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_steps1">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_goto1">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_goto2">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_gotoX">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1920</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_gotoY">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1080</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>77</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_motorgo1">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_motorgo2">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_motorgoto1">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_motorgoto2">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_motorgotoX">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_motorgotoY">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Motor Setting</string>
      </property>
      <widget class="QPushButton" name="btn_setparam">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>[Motor 1] rpm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>[Motor 1] Steps / rev</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>[Motor 2] rpm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>[Motor 2] Steps / rev</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QSpinBox" name="spb_rpm1">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_stepperrev1">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_rpm2">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spb_stepperrev2">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Enconder</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>16</y>
         <width>114</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSpinBox" name="spb_origin2">
          <property name="minimum">
           <number>-2000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spb_origin1">
          <property name="minimum">
           <number>-2000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spb_corner2">
          <property name="minimum">
           <number>-2000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>590</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spb_corner1">
          <property name="minimum">
           <number>-2000</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>330</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="spb_currentX_pixel">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1920</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spb_currentY_pixel">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1080</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>16</y>
         <width>68</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>screen origin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>screen corner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Current XY</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>198</x>
         <y>16</y>
         <width>77</width>
         <height>83</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QPushButton" name="btn_setOrigin">
          <property name="text">
           <string>Set origin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setCorner">
          <property name="text">
           <string>Set corner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_autosetEncoder">
          <property name="text">
           <string>Autoset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Motor Monitor</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>motor1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_posmotor1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>motor2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_posmotor2">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Macro</string>
      </property>
      <widget class="QPushButton" name="btn_auto61">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto 6-1</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
